src/containers/pages/InterventionPlan/PlanDefinitionList/index.tsx

```diff
@@ -143,8 +150,13 @@ interface DispatchedStateProps {
 }
 
 /** map state to props */
-const mapStateToProps = (state: Partial<Store>): DispatchedStateProps => {
-  const planDefinitionsArray = getPlanDefinitionsArray(state);
+const mapStateToProps = (state: Partial<Store>, ownProps: any): DispatchedStateProps => {
+  const searchString = trimStart(ownProps.location.search, '?');
+  const queryParams = querystring.parse(searchString);
+  const searchedTitle = queryParams.search as string;
+  const planDefinitionsArray = !searchedTitle
+    ? getPlanDefinitionsArray(state)
+    : getPlanDefinitionsArrayByTitle()(store.getState(), { title: searchedTitle });
```

Comment:
Doesn't seem like this condition is required. IMO i.e. seems like it's already handled in the selector